script.js

```diff
@@ -0,0 +1,107 @@
+// "use strict";
+
+// ES5
+function Calculator(value) {
+	if (typeof(value) == 'number') {
+		this.int = value;
+	} else {
+		this.str = value

```

Comment:
Do we really need separate variables for storing the string or number? Why not just change this to `value` or something